package.json

```
@@ -1,91 +1,76 @@
 {
   "name": "@gatsbyjs/reach-router",
-  "version": "1.3.9",
-  "description": "Next generation Routing for React.",
-  "main": "index.js",
-  "module": "es/index.js",
+  "version": "1.3.10-no-navigate.5",
```

We probably should not change version in this PR - the versioning should be separate commit (and it would "premajor" bump

```suggestion
  "version": "1.3.9",
```